### PR TITLE
Issue 288 - Glowing extension icon

### PR DESCRIPTION
**Pull request by danielgoldelman**

### CODE REVIEW

**Comment by notowen333** [comment]
The code looks good to me! 

What's the thinking behind 30 for the evidence threshold? Maybe add this explanation as a comment in `constants.js`

Obviously, we'll need an updated SVG. @Lr-Brown should push this updated SVG to this branch before merge.

File: src/background/analysis/constants.js [hunk 1]
```diff
@@ -10,8 +10,18 @@ export const MAX_BYTE_LEN = 200000
  */
 export const MAX_CHAR_LEN = 100000
 
+/**
+ * Used in setTimeout when changing favicon
+ */
+export const FIVE_SEC_IN_MILLIS = 5000
+
+/**
+ * Used to see if evidence is large for a first party
```
Comment (notowen333) [inline src/background/analysis/constants.js:19]: Probably mention that this is used to make a decision about the rocketship svg here.

File: src/background/background.js [hunk 2]
```diff
@@ -50,6 +52,42 @@ browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
...
+    } else {
+      // Change it back to the original. Sometimes quickly changing a webpage after load
+      // keeps the now-incorrect favicon
+      browser.browserAction.setIcon({tabId: currentWindowId, path:"../assets/favicon.svg"})
+    }
+  }
+  // timeout set to 5 seconds to allow for initial third parties loaded.
+  setTimeout(swapFavicon, FIVE_SEC_IN_MILLIS)
+}
+browser.webNavigation.onDOMContentLoaded.addListener(changeFavicon)
```
Comment (notowen333) [inline src/background/background.js:89]: Add a comment highlighting that this is opening a listener and calling the above function.

**Review by notowen333 — APPROVED** [review]
Great PR. The code is clean and easy to follow.

**Comment by notowen333** [comment]
The rocket now "takes off" pretty well. Run the branch to see how it looks at tiny scale.
Screen Shot 2021-09-23 at 3 37 10 PM
.

**Comment by SebastianZimmeck** [comment]
Nice! As far as I am concerned, feel free to merge @notowen333 or @danielgoldelman.

**Pull request merged**